ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -88,9 +88,13 @@
   [concept]
   (if-errors-throw :bad-request
                    (if (mt/umm-json? (:format concept))
-                     (umm-spec/validate-metadata (:concept-type concept)
-                                                 (:format concept)
-                                                 (:metadata concept))
+                     (let [umm-version (mt/version-of (:format concept))
+                           accept-version (config/ingest-accept-umm-version)]
+                       (if (>= 0 (compare umm-version accept-version))
+                         (umm-spec/validate-metadata (:concept-type concept)
+                                                     (:format concept)
+                                                     (:metadata concept))
+                         [(str "umm version is higher than ingest-accept-umm-version" (pr-str accept-version))]))
```

Comment:
I thought that we could give them a better error message if we could tell if their version is valid. If that's not possible here, then we still need to give them a better error message.

Maybe something like "UMM JSON version X or lower can be ingested. Any version above that is considered in-development and cannot be ingested at this time." I want to convey why they can't ingest what they may think would be a valid version.